libs/ngworker/lumberjack/schematics/ng-add/schema.json

```
@@ -0,0 +1,39 @@
+{
+  "$schema": "http://json-schema.org/schema",
+  "id": "SchematicsNgWorker",
+  "title": "Ng Worker add schematics",
+  "type": "object",
+  "properties": {
+    "project": {
+      "type": "string",
+      "description": "The name of the project.",
+      "aliases": ["p"],
+      "$default": {
+        "$source": "projectName"
+      }
```

How does this work? Shouldn't it be:
```suggestion
      "$default": {
        "$source": "argv",
        "index": 0
      },
```

---

It refers to the default project